#
# Copyright (c) 2015, 2016, Oracle and/or its affiliates. All rights reserved.
#
Prevent access to uninitialized memory in udat_getSymbols
https://unicode-org.atlassian.net/browse/ICU-12375

--- icu/source/i18n/udat.cpp.orig
+++ icu/source/i18n/udat.cpp
@@ -745,6 +745,10 @@
         res = syms->getZodiacNames(count, DateFormatSymbols::FORMAT, DateFormatSymbols::NARROW);
         break;
 
+    default:
+	*status = U_UNSUPPORTED_ERROR;
+	return 0;
+
     }
 
     if(index < count) {
